### DISCUSSION TITLE
Facing issues while starting docker based quorum node

### DISCUSSION THREAD

**Discussion by tkb2501** [ConsenSys/quorum#1650] [General]
Hi Team,

```
Fatal: Failed to read genesis file: open /data/qbft-standard-genesis.json: no such file or directory
Fatal: Failed to read genesis file: open /data/qbft-standard-genesis.json: no such file or directory
```

We are facing some issue while starting a new quorum node using docker file. It might be because of some file permissions can someone please help with the issue, as we have already provided the required permissions to the data directory and the genesis file.

**Comment by baptiste-b-pegasys** [comment]
Can you provide more information : where comes the docker file, do you use entry-point script, etc.
